src/styles/main.scss

```diff
@@ -1,3 +1,63 @@
+@import "./utils/variables.scss";
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  font-family: "Arial", sans-serif;
+  font-size: 30px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+
+  width: $calendar-width;
+  padding-inline: $padding-inline-size;
+  gap: $gap-size;
+
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+    background-color: $cell-color;
+
+    border: solid 1px black;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $hovered-cell-color;
+      transform: translateY(-20px);
+      transition: 0.5s;
+    }
```

Comment:
these properties should be implemented for the whole element not only for the effect

```suggestion
    align-items: center;
    transition: 0.5s;
    cursor: pointer;

    &:hover {
      background-color: $hovered-cell-color;
      transform: translateY(-20px);
    }
```